liquibase/changeLogs/capture/tables/jsonData/changeLog.yml

```diff
@@ -105,3 +105,18 @@ databaseChangeLog:
       changes:
         - sql: create table if not exists ${AQTS_SCHEMA_NAME}.json_data_december partition of ${AQTS_SCHEMA_NAME}.json_data for values in (12);
         - rollback: drop table if exists ${AQTS_SCHEMA_NAME}.json_data_december;
+
+  - changeSet:
+      author: kkehl
+      id: "alter.table.${AQTS_SCHEMA_NAME}.json_data.uuid"
+      preConditions:
+        - onFail: MARK_RAN
+        - onError: HALT
+        - not:
+            - columnExists:
+                schemaName: ${AQTS_SCHEMA_NAME}
+                tableName: json_data
+                columnName: uuid
+      changes:
+        - sql: alter table ${AQTS_SCHEMA_NAME}.json_data add column if not exists uuid character(36);
```

Comment:
Are these always 36 characters long? Just curious.

Reply:
Think so.  The UUID4 format looks like:  16fd2706-8baf-433b-82eb-8c7fada847da.

Also, the example I found in S3 looks like:

body_getTSData_10000_mod_    +   0074973a-c377-472f-9f3c-f093fdc18836  + .json